.pre-commit-config.yaml

```diff
@@ -9,6 +9,11 @@ repos:
   - id: flake8
 - repo: https://github.com/pycqa/isort
   rev: 5.10.1
-    hooks:
-    - id: isort
-      name: isort (python)
+  hooks:
+  - id: isort
```

Comment:
@igortg do you feel strongly about `isort`? I personally have been using [reorder-python-imports](https://pypi.org/project/reorder-python-imports/) in all my repos.

Reply:
Better... @LucasRochaAbraao will do the change.

Reply:
Changed it.